zebra/zebra_nhg.c

```diff
@@ -3028,11 +3028,30 @@ static void zebra_nhg_sweep_entry(struct hash_bucket *bucket, void *arg)
 	 */
 	if (ZEBRA_NHG_CREATED(nhe) && nhe->refcnt <= 0)
 		zebra_nhg_uninstall_kernel(nhe);
+
+	return HASHWALK_CONTINUE;
```

Comment:
isn't this also a path where multiple nhes could be freed - when `zebra_nhg_handle_uninstall()` runs?

Reply:
yes but that process goes through the dplane.  Not much to worry about there since it won't be in the context of this current iteration.

Reply:
I missread.  Fixing